<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>260</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>290</width>
    <height>260</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>290</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Добавяне на контролер</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {border-color: rgb(0, 0, 0);border-style: solid; border-width: 1px;}</string>
   </property>
   <property name="title">
    <string>Данни за контролера</string>
   </property>
   <widget class="QLineEdit" name="lineName">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Име</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineIPAddress">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>000.000.000.000;_</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Адрес</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="linePort">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>000;_</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Порт</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>53</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Парола</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Повторете парола</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineNewPassword">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineNewPasswordRepeat">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Запис</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
